.gitmodules

```diff
@@ -7,3 +7,6 @@
 [submodule "third_party/json"]
 	path = third_party/json
 	url = https://github.com/nlohmann/json.git
+[submodule "third_party/utfcpp"]
```

Comment:
Can you add a section to [ThirdPartyNotices.txt](https://github.com/ni/grpc-device/blob/main/ThirdPartyNotices.txt) for this in a similar fashion to what was added for grpc and json?

Reply:
Done